prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'JunLee';
+    console.log(fullName);
+    var numberOfPets = 1;
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = 'null';
+    console.log(nothing);
+    var never = 'var';
```

Comment:
To declare a variable with no value, you can just use the `var` keyword and the variable name.

```js
var never;
```

Reply:
![javascript variables changed](https://user-images.githubusercontent.com/88108910/128799420-96d36cd8-b2d9-4db2-ab7d-9f1de51ed202.png)